web/packages/teleterm/src/sharedProcess/ptyHost/ptyProcess.test.ts

```diff
@@ -52,4 +52,29 @@ describe('PtyProcess', () => {
       );
     });
   });
+
+  if (process.platform !== 'win32') {
```

Comment:
Kinda cowboyish way to avoid running this test on Windows. cmd.exe doesn't have its Ctrl+D equivalent and even if sh.exe is installed, it doesn't seem to work the same way on Windows.